Makefile

```diff
@@ -16,4 +19,7 @@ sd2snesrbo:
 sd2sneskpdr25:
 	cp resources/sm_orig.sfc build/smhack20_sd2snes_kpdr25.sfc && cd src && asar --no-title-check -DFEATURE_SD2SNES=1 -DCATEGORY=2 main.asm ../build/smhack20_sd2snes_kpdr25.sfc && cd -
 
-all: sd2snes smhack sd2snesrbo smhackrbo sd2sneskpdr25 smhackkpdr25
+sd2snesgtclassic:
+	cp resources/sm_orig.sfc build/smhack20_sd2snes_gtclassic.sfc && cd src && asar --no-title-check -DFEATURE_SD2SNES=1 -DCATEGORY=3 main.asm ../build/smhack20_sd2snes_gtclassic.sfc && cd -
+
+all: sd2snes smhack sd2snesrbo smhackrbo sd2sneskpdr25 smhackkpdr25 sd2snesgtclassic shmackgtclassic
```

Comment:
"shmackgtclassic" should be "smhackgtclassic".

Reply:
Good catch. Resolved now.